templates/html/header.tmpl

```diff
@@ -264,10 +264,10 @@ fieldset:disabled .btn {
 <body>
 <div class="container">
   <div class="row">
-  <div class="col-logo"><a href="www.cyverse.org" target="_blank"><img class="icons" src="https://cyverse.org/sites/default/files/cyverse_logo.png" alt="cyverse logo" /></a></div>
+  <div class="col-logo"><a href="www.cyverse.org" target="_blank"><img class="icons" src="https://cyverse.org/sites/default/files/cyverse_logo.png" alt="CyVerse" /></a></div>
   <div class="spacer"></div>
   <div class="col-title">
-    <a href="https://de.cyverse.org/" target="_blank"><img class="logo" src="https://cyverse.org/sites/default/files/inline-images/de_icon.png" alt="de logo"/></a>
+    <a href="https://de.cyverse.org/" target="_blank"><img class="logo" src="https://cyverse.org/sites/default/files/inline-images/de_icon.png" alt="DE"/></a>
```

Comment:
These alt text changes are because they're shown in the plaintext email. With this change, the top of the email ends up like:

```
CyVerse ( www.cyverse.org )
DE ( https://de.cyverse.org/ ) Discovery Environment
```
where before, it would say 'cyverse logo' instead of 'CyVerse', and 'de logo' instead of 'DE'. I think these are still reasonable alt texts for these icons, hopefully.